### DISCUSSION TITLE
Shapeile for MISO Region

### DISCUSSION THREAD

**Discussion by shivamparashari** [electricitymaps/electricitymaps-contrib#4855] [Q&A] [answered]
Hi,

I am working on MISO region electricity consumption, congestion and transmission flow for a simulation model. For that I am looking for the shapefile/GDB/JSON/GeoJSON/XML file that would contain:
 - MISO area
 - Zones area
 - County area
 - Attribute information

The map ArcGIS online map [here ](https://www.arcgis.com/home/webmap/viewer.html?webmap=8e32111ac8964e8a82f39bdc3fea347d)displays MISO region and the counties, but it is not in the downloadable form. Also, the MISO region is not exactly as per the administrative area divisions. 

Can anyone help me with the information?

![image](https://user-images.githubusercontent.com/16076305/207242410-8d97b040-f20f-420b-8577-fb12d8298caf.png)
MISO Region

Thank you.

**Comment by VIKTORVAV99** [comment, marked as answer]
We only have the geometries that are in `web/geo/world.geojson` but I don't know if that is sufficient for your needs.

**Reply by shivamparashari** [reply]
Hey thank you @VIKTORVAV99 for your reply. I was able to find the required data from HIFLD [website](https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::independent-system-operators-1/explore).
